src/lib.rs

```diff
@@ -33,15 +33,14 @@
 //!                 .skip_while(|l| !l.starts_with(COMMENT_PREFIX))
 //!                 // Extract consecutive commented lines.
 //!                 .take_while(|l| l.starts_with(COMMENT_PREFIX))
-//!                 // Strip the initial "//" from commented lines.
```

Comment:
Did we mean to remove this line?

Reply:
Yeah I've more-or-less just moved README.md into lib.rs.